src/components/common/Description.js

```diff
@@ -10,20 +10,30 @@ const Description = ({ title, content, color, images, labels }) => (
         horizontal: 'xlarge',
         bottom: 'medium',
       }}
+      align=""
```

Comment:
what's this?

Reply:
Unused code I must have forgotten to remove. It is removed now

Reply:
@JoeButler7 A good practice, you can benefit from in the long run, will be to have your own review done before assigning reviewers, I'm always surprised by the little tweaks I missed once reviewing my own code.